src/summarycode/plugin_consolidate.py

```diff
@@ -383,15 +386,19 @@ def create_consolidated_components(resource, codebase, holder_key):
     given resource and holder key
     """
     license_expressions = []
+    unknwown_expressions = []
```

Comment:
`unknwown_expressions` should be `unknown_expressions`

Reply:
Oops! typo